sunpy/net/vso/attrs.py

```diff
@@ -400,7 +400,12 @@ class Quicklook(_VSOSimpleAttr):
     Quicklook products are *not* searched by default.   Reference:
     documentation in SSWIDL routine vso_search.pro.
     """
-    pass
+    def __init__(self,value):
+        super(Quicklook,self).__init__(value)
+        if(self.value):
```

Comment:
yeah, this is what i get for not being able to code.

Back into the vscode mines.